modules/unauthenticated-users-cluster-role-binding-con.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * authentication/using-rbac.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="unauthenticated-users-cluster-role-bindings-concept_{context}"]
+= `ClusterRoleBindings` for unauthenticated groups
+
+{product-title} {product-version} does not allow unauthenticated groups access to `ClusterRoleBindings` for security.
+
```

Comment:
Since the below stuff is talking about past release details, I wonder whether it would make sense to put this information in a note? Because right now it jumps a little - first para talks about current behavior, next para talks about past behavior, then third para goes back to talking about current behavior and if you need to add this access back.

Not sure if this is the best idea, but just thinking out loud - no need to do it if you prefer the current way

Reply:
I agree that a note would be better. I don't love putting 2 callouts in the same module but it does seem to make sense.

Reply:
Maybe we could write the docs for `cicd/builds/triggering-builds-build-hooks.adoc`, like it was always there in the first place and not like this is a recent change.

But we should definitely have a spot, where we document the change. If I would be an administrator that observes a regression a behavior, where would I take a look? The release notes?

Reply:
I think we should at least mention it here, but it will be covered more in the release note.

Reply:
I tend to agree that a NOTE might make the flow better. Although, I also agree that maybe it doesn't belong here. I tend to avoid version-specifics in the docs. Leave it for release notes.